src/gov/nist/javax/sip/parser/PipelinedMsgParser.java

```diff
@@ -459,7 +460,14 @@ public void run() {
                     if (stackLogger.isLoggingEnabled(StackLogger.TRACE_DEBUG)) {
                         stackLogger.logDebug("About to parse : " + inputBuffer.toString());
                     }
-                    sipMessage = smp.parseSIPMessage(inputBuffer.toString().getBytes(), false, false, sipMessageListener);
+                    byte[] inputBufferBytes;
+                    try {
+                        inputBufferBytes = inputBuffer.toString().getBytes("UTF-8");
+                    } catch (UnsupportedEncodingException e) {
+                        // log the lack of UTF8 support here?
```

Comment:
i think we should log this under WARNING level, this means an incorrect JVM configuration...